test/bidirectional-streams.spec.js

```diff
@@ -0,0 +1,118 @@
+import { createServer } from './fixtures/server.js'
+import { getReaderValue } from './fixtures/reader-value.js'
+import { WebTransport } from '../lib/index.js'
+import { expect } from 'chai'
+import { readStream } from './fixtures/read-stream.js'
+import { writeStream } from './fixtures/write-stream.js'
+import { defer } from '../lib/utils.js'
+
+/**
+ * @template T
+ * @typedef {import('../lib/types').Deferred<T>} Deferred<T>
+ */
+
+const SERVER_PATH = '/unidirectional-streams'
+
+describe('bidirectional streams', function () {
```

Comment:
Linter complains here and on other lines, that describe is not defined...., I am also puzzled from where the definition should come. One can disable it globally for the file, but normally it is very handy

Reply:
Mocha injects these into the global scope, with eslint you can put a comment at the top of the file to tell it the file is a mocha test:

```js
/* eslint-env mocha */
```